src/v/cluster/health_monitor_backend.cc

```diff
@@ -635,6 +615,37 @@ ss::future<std::error_code> health_monitor_backend::collect_cluster_health() {
         }
     }
     _reports_disk_health = cluster_disk_health;
+
+    if (config::shard_local_cfg().enable_usage()) {
+        vlog(clusterlog.info, "collecting cloud health statistics");
+
+        cluster::cloud_storage_size_reducer reducer(
+          _topic_table,
+          _members,
+          _partition_leaders_table,
+          _connections,
+          topic_table_partition_generator::default_batch_size,
+          cloud_storage_size_reducer::default_retries_allowed);
+
+        try {
```

Comment:
@alenkacz thinking about the -1 returned for the leader requirement again, why is this a requirement? We can just have all nodes report what they see in the cluster metadata. I know you will probably be asking me that you'd want the freshest of the reported values, but even in the case where the only the leader  reports one, it will still be stale by some amount.

Reply:
This was per rfc, we can change it but if all report it, how do I choose which one to pick? Random? For open bucket they won't be the same, right?

Reply:
They won't your right but they will all be up to date within a few seconds with respect to the configured health metadata refresh interval. Think about the alternative though, if only one node reports this value (and followers return -1) then if at window close theres a leadership change, all nodes will report -1 (none are leader) and this value for the entire hour is lost

Reply:
Do you folks want to jump on a call to align quicker, perhaps?

Reply:
@graphcareful alright, let's do that. I'll update the spec later today to reflect this. 

maybe one last question: will the value be the same on all brokers for closed windows?

sure, let's have a call today if still needed 👍